pod/enrichment/templates/enrichment/edit_enrichment.html

```diff
@@ -20,68 +16,86 @@
   /*** For the form ***/;
   var video_duration = {{video.duration}};
   /*** For the Cancel button ***/
-  $(document).on("reset", "#page-video form", function (e) {
-    $("span#form_enrich").html("");
-    $("form#form_new").show();
-    $("form").show();
-    $("table tr").removeClass("info");
+ 
+  document.addEventListener("reset",   (e) => {
+    if (e.target !== document.querySelector("#page-video form"))  return
+    document.querySelector("span#form_enrich").html = "";
+    document.getElementById("form_new").style.display = "block";
+   document.querySelectorAll("form").style.display = "block";
```

Comment:
À mon avis, le `document.getElementById("form_new").style.display = "block";`est inutile, car le `document.querySelectorAll("form").style.display = "block";` doit déjà inclure form_new

Reply:
nb : il n'y a pas besoin d'un "for each" sur le querySelectorAll ?

Reply:
ou alors c'est que seul le form_new est utile ^^.

Reply:
nb2 : vu que les id sont uniques, plutot que querySelector("span#form_enrich") on doit aussi pouvoir faire getElementById("form_enrich")